test/extended/apiserverauth/api.go

```diff
@@ -0,0 +1,57 @@
+package apiserverauth
+
+import (
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/tidwall/gjson"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-API_Server]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithoutNamespace("kcp-api")
+	)
+
+	g.It("Author:knarra-Medium-1001-[Smoke] Checking apiserver version should display correctly", func() {
+		g.By("# Check the apiserver version display correctly")
```

Comment:
Please help change this line to have something like "check cps version gets displayed correctly"

Reply:
Sure. Thanks!